content/it/docs/reference/glossary/job.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Job
+id: job
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/jobs-run-to-completion
+short_description: >
+  Uno o più lavori(task) che vengono eseguiti fino al loro completamento.
```

Comment:
before an opening parentheses "(" we need a space, so s/lavori(task)/lavori (task)/